bigchaindb/common/transaction.py

```diff
@@ -1248,17 +1248,14 @@ def __str__(self):
         tx = Transaction._remove_signatures(self.to_dict())
         return Transaction._to_str(tx)
 
-    @classmethod
-    # TODO: Make this method more pretty
-    def from_dict(cls, tx_body):
-        """Transforms a Python dictionary to a Transaction object.
+    @staticmethod
+    def validate_structure(tx_body):
```

Comment:
`validate_structure` as it is... It is validating against the schema and also checking the ID.